pkg/webhook/policy.go

```diff
@@ -68,6 +74,7 @@ var (
 	deserializer                       = codecs.UniversalDeserializer()
 	disableEnforcementActionValidation = flag.Bool("disable-enforcementaction-validation", false, "disable validation of the enforcementAction field of a constraint")
 	logDenies                          = flag.Bool("log-denies", false, "log detailed info on each deny")
+	emitDenyEvents                     = flag.Bool("emit-deny-events", false, "emit Kubernetes events in gatekeeper namespace with detailed info on each deny")
```

Comment:
`emit-deny-events` flag is false here but it is set in the yaml

Reply:
yea we need the flag to be set for e2e